terms/collections.md

```diff
@@ -1,7 +1,12 @@
 # Semantics
 
-Agreed upon, often hierarchical, [vocabulary](change-requests.md).
+Agreed upon, often hierarchical, set of terms and structures to descibe data. This includes [vocabulary](vocabulary.md), [schema](schema.md), [taxonomies][taxonomy.md], and [ontologies][ontology.md]
 
-A way to interpret signs and symbols (to confer them sense and meaning) within particular context. Semantics relies on the rules of [syntax](the-gitbook-editor.md), for this interpretation.
+A way to interpret signs and symbols (to confer them sense and meaning) within particular context. Semantics relies on the rules of [syntax](syntax.md), for machine readability.
```

Comment:
There are a few cases where files aren't named correctly, I'll come back and update their names and references to be consistent throughout. In the meantime, please don't change hyperlinks from working to broken!

Reply:
OK. These names are confusing me when I make commits to forks.

Reply:
Exactly. They’re all fixed on main, so update your fork and you’ll be good.

honestly though it’s much easier for me if you use the editor, because syntax problems are very hard to debug.